OpenRA.Game/FileFormats/VocLoader.cs

```diff
@@ -0,0 +1,348 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OpenRA.FileFormats
+{
+	public class VocLoader : Stream
```

Comment:
When we do eventually move to streaming I think it would be better to have a method on the reader that returns a stream instead of making the reader itself a stream.  This will cut down on boilerplate for the cases where we already have the raw data as a stream (e.g. loading aud from a mix file) and can return that directly.

Thoughts?